### DISCUSSION TITLE
Broken link: Github's own link to "Share feedback and report bugs" for lists is incorrect

### DISCUSSION THREAD

**Discussion by varenc** [community/community#43168] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The incorrect link appears at the bottom of the "Create List" modal.  The modal is shown when you click "Create list" in any repo's star indicator dropdown.

<img width="507" alt="image" src="https://user-images.githubusercontent.com/167585/210295299-6d3f0cb7-6eff-409f-865f-16e592b6e9f4.png">

Specifically the text "_Share feedback and report bugs._" links to the wrong category with no results: https://github.com/community/community/discussions/categories/lists-feedback

It links to `list-feedback` instead of just `lists`. I believe it should link here instead: https://github.com/community/community/discussions/categories/lists

**Comment by radar-blip** [comment]
Maybe a bug report is in order. 
You can file a bug report if you want, here: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by Nwekoyo** [reply]
The connection to this service critical
